### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by jackmaloney**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -189,11 +189,14 @@ dependencies {
...
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '3.0.0'
-  compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '3.0.74'
+  compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '3.0.75'
 
   //Remove when our dependencies pull in this version or later
-  compile 'com.fasterxml.jackson.core:jackson-databind:2.9.9.3'
-  integrationTestCompile 'com.fasterxml.jackson.core:jackson-databind:2.9.9.3'
+  compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.9.10'
+  integrationTestCompile 'com.fasterxml.jackson.core:jackson-databind:2.9.10'
```
Comment (doncem) [inline build.gradle]: this line is obsolete due to `integrationTestCompile sourceSets.main.runtimeClasspath`

File: build.gradle [hunk 2]
```diff
@@ -189,11 +189,14 @@ dependencies {
...
+
+  compile group: 'com.microsoft.azure', name: 'applicationinsights-logging-logback', version: '2.4.1'
+  compile group: 'com.microsoft.azure', name: 'applicationinsights-spring-boot-starter', version: '2.4.1'
```
Comment (doncem) [inline build.gradle]: perhaps want to consider to bump agent too? this will require agent jar and `web.xml`

File: build.gradle [hunk 3]
```diff
@@ -236,6 +236,10 @@ dependencyManagement {
       entry 'tomcat-embed-websocket'
     }
 
+    dependencySet(group: 'com.fasterxml.jackson.core', version: '2.9.10') {
+      entry 'jackson-databind'
+    }
+
```
Comment (doncem) [inline build.gradle]: Then there is no need to include `testCompile` ;)

File: build.gradle [hunk 4]
```diff
@@ -236,6 +236,10 @@ dependencyManagement {
       entry 'tomcat-embed-websocket'
     }
```
Comment (doncem) [inline build.gradle]: we can remove whole lot block for tomcat - it has been fixed in previous versions of spring boot already

File: build.gradle (hunk 3 above)
Comment (jackmaloney) [reply build.gradle]: testCompile where @doncem ?

Comment (ryandac) [reply build.gradle]: we were trying to fix the security vulnerabilities. 
> One or more dependencies were identified with known vulnerabilities in sscs-bulk-scan:

>gradle-pitest-plugin-1.4.5.jar (cpe:2.3:a:gradle:gradle:1.4.5:*:*:*:*:*:*:*) : CVE-2019-11065, CVE-2019-15052, CVE-2019-16370
>wiremock-standalone-2.20.0.jar/META-INF/maven/com.fasterxml.jackson.core/jackson-databind/pom.xml (pkg:maven/com.fasterxml.jackson.core/jackson-databind@2.8.11.2, cpe:2.3:a:fasterxml:jackson:2.8.11.2:*:*:*:*:*:*:*, cpe:2.3:a:fasterxml:jackson-databind:2.8.11.2:*:*:*:*:*:*:*) : CVE-2019-14540, CVE-2019-16335 

We've upgraded pitest plugin to the latest and forced to use the latest jackson-databind. Although v 2.8.11.2 is not pulled in, `dependencyCheckAnalyze` still complains.

Comment (doncem) [reply build.gradle]: I'm afraid you cannot bump that for wiremock :(. Will have to be suppressed. Or try use `2.25.0` I believe it is all good now with wiremock on that version. Unless I'm mixing with something else nowadays.

**Pull request merged**